.buildkite/scripts/integration-tests.ps1

```diff
@@ -2,21 +2,11 @@ param (
     [string]$GROUP_NAME
 )
 
-Write-Output "~~~ Switching to PowerShell 7"
-pwsh
-$PSVersionTable.PSVersion
+echo "~~~ Preparing environment"
 
-# TODO: dedicated ESS strack for retries
-Write-Output "~~~ Receiving ESS stack metadata"
-$env:ELASTICSEARCH_HOST = (buildkite-agent meta-data get "es.host")
-$env:ELASTICSEARCH_USERNAME = (buildkite-agent meta-data get "es.username")
-$env:ELASTICSEARCH_PASSWORD = (buildkite-agent meta-data get "es.pwd")
-$env:KIBANA_HOST = (buildkite-agent meta-data get "kibana.host")
-$env:KIBANA_USERNAME = (buildkite-agent meta-data get "kibana.username")
-$env:KIBANA_PASSWORD = (buildkite-agent meta-data get "kibana.pwd")
+$PSVersionTable.PSVersion
 
-Write-Output "~~~ Running integration tests as $env:USERNAME"
-Write-Output "~~~ Integration tests: $GROUP_NAME"
+. "$PWD\.buildkite\scripts\steps\ess.ps1"
```

Comment:
`.` and `$PWD` both imply the current directory, which shouldn't need to be specified if we use a relative path? Can we just do `.buildkite\scripts\steps\ess.ps1` instead?

Reply:
I was wrong. I misunderstood this call. https://stackoverflow.com/questions/6016436/in-powershell-how-do-i-define-a-function-in-a-file-and-call-it-from-the-powersh

Resolving the comment